dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import print_function
+
+import os
+import unittest
+from cgi import test
+
+import networkx as nx
+
+from dataprofiler.profilers.graph_profiler import GraphProfile
+
+test_root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestGraphProfiler(unittest.TestCase):
+    def setUp(self):
+        self.graph = nx.Graph()
+        self.graph.add_nodes_from([1, 2, 3, 4])
+        self.graph.add_edges_from(
+            [
+                (1, 2, {"id": 1, "weight": 2.5}),
+                (2, 3, {"id": 2, "weight": 1.7}),
+                (3, 4, {"id": 3, "weight": 1.8}),
+                (4, 1, {"id": 4, "weight": 4.1}),
+                (1, 3, {"id": 4, "weight": 2.1}),
+            ]
+        )
+        self.graph_profile = GraphProfile("graph")
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestGraphProfiler, cls).setUpClass()
+
+    # WIP
+    def test_init(self):
+        self.assertTrue(True, True)
+
+    # WIP
+    def test_diff(self):
```

Comment:
sounds weird, but maybe just validate not implemented and that we raise the expected error str